src/storage/logs.js

```diff
@@ -0,0 +1,44 @@
+const db = require('./DataAccess');
+const { LOGS, ERRORS, REQUESTS } = require('./collections');
+const debugConsole = require('debug');
+const { v4: uuidv4 } = require('uuid');
+
+function debug(location) {
+  return message => {
+    const logger = debugConsole(location);
+    logger(message);
+    const log = {
+      id: uuidv4(),
+      timestamp: Date.now(),
+      message: message,
+      location: location
+    };
+    db.insert(LOGS, log);
+  };
+}
+
+function error(location) {
+  return error => {
+    const logger = debugConsole(location);
+    logger(error);
+    const log = {
+      id: uuidv4(),
+      timestamp: Date.now(),
+      error: error,
+      location: location
+    };
+    db.insert(ERRORS, log);
+  };
+}
+
+function storeRequest(request) {
+  const log = {
```

Comment:
If there was a bearer token associated with it can we store the clientId (or just 'admin' if using the admin token)